pajbot/modules/masspoints.py

```diff
@@ -21,7 +20,7 @@ class MassPointsModule(BaseModule):
     SETTINGS = [
         ModuleSetting(
             key="sub_points",
-            label="Points to give for subscribers for each pleb point",
+            label="Award this amount of points to subs for every pleb point",
```

Comment:
Just make it "multiply points given to subscribers by this factor", sorry for not directly suggesting what I had in mind earlier

Reply:
Additionally, now I realize this, but I think it would be nice to have this as a "percentage" setting where a setting of 100 gives plebs and subs the same amount. E.g. 200 would give subs twice the amount of points, etc. That way you can set it to something other than whole number multipliers.